### ISSUE TITLE
Magmatic Generator from Extra Utilities ain't working..

### ISSUE THREAD

**Issue by CryptKing6626070** [FTBTeam/FTB-Presents-Stoneblock-2#770]
![Minecraft 1 12 2 02_08_2020 18_47_18](https://user-images.githubusercontent.com/69116189/89126755-ec4e1080-d4f0-11ea-9f2d-6db3dc3e4b60.png)

The Magmatic Generator isn't working...it's description says u need lava to make energy...if the screenshot I've attached works you will see that it clearly doesn't work with lava nor water. I made a whole process for it: cobblestone generator tier 5->chest->transfer node(item)->transfer pipe->crucible->lava->transfer node(fluid)->transfer pipe->magmatic generator(has full lava).

I have no power from the generators..what can i do?

**Comment by ToxicCosmos** [comment]
Not sure why your generator isn't working, but here's my setup that is: T1 cobblestone generator -> itemduct with servo (thermal dynamics) -> crucible -> hardened fluiduct with servo (thermal dynamics) -> magmatic generator

**Comment by ToxicCosmos** [comment]
@CryptKing6626070 I think I've figured it out. I found myself with the same issue when I tried to add a speed upgrade to my magmatic generator. After some digging, it turns out that if you get the error "Grid is overloaded" (by hovering over the crossed-out arrow), it's because you don't have enough Grid Power. I can see in your screenshot (under the inventory) that you indeed don't have any GP available. You'll have to create some GP mills from Extra Utilities 2 such as the Lava Mill, Water Mill, Wind Mill, or Fire Mill. Hope this helps!
